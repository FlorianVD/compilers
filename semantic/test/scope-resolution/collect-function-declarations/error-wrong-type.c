// RUN-WITH-ARGS: -dump-function-table

foobar f() {}
