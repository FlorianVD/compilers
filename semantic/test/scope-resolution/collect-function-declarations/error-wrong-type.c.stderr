sema: error: 3:1: Unknown type 'foobar'
